algebra/ring.lean

```diff
@@ -57,6 +57,151 @@ section comm_ring
   ⟨dvd_of_neg_dvd, neg_dvd_of_dvd⟩
 end comm_ring
 
+def is_unit {α : Type u} [comm_ring α] (x : α) := ∃ y, x * y = 1
+def nonunits (α : Type u) [comm_ring α] : set α := { x | ¬∃ y, x * y = 1 }
+
+class is_hom {α : Type u} {β : Type v} [comm_ring α] [comm_ring β] (f : α → β) : Prop :=
+(map_add : ∀ {x y}, f (x + y) = f x + f y)
+(map_mul : ∀ {x y}, f (x * y) = f x * f y)
+(map_one : f 1 = 1)
```

Comment:
Change the name to `is_ring_hom`. Don't forget you're in the global namespace here.